wordle-web/src/views/GameView.vue

```diff
@@ -0,0 +1,34 @@
+<template>
+    <div class="game">
+        <v-card>
+          <v-card-title>
+              <h1>Word War 1</h1>
```

Comment:
No need to wrap this in an h1 tag, if the sizing for your v-card-title isn't to your liking, you can use the typography classes from vuetify.